AppInspector.CLI/Program.cs

```diff
@@ -263,6 +263,31 @@ private static void ValidFileWritePath(string filePath)
 
         #region RunCmdsWriteResults
 
+        private static int RunGetTagsCommand(CLIGetTagsCommandOptions cliOptions)
+        {
+            GetTagsResult.ExitCode exitCode = GetTagsResult.ExitCode.CriticalError;
+
+            GetTagsCommand command = new GetTagsCommand(new GetTagsCommandOptions()
+            {
+                SourcePath = cliOptions.SourcePath ?? "",
+                CustomRulesPath = cliOptions.CustomRulesPath ?? "",
+                IgnoreDefaultRules = cliOptions.IgnoreDefaultRules,
+                ConfidenceFilters = cliOptions.ConfidenceFilters,
+                FilePathExclusions = cliOptions.FilePathExclusions,
```

Comment:
Does FilePathExclusions include extension filters? Current tool has an option to filter out specific extensions

Reply:
File path exclusions does a .contains in the things listed as arguments. I didn't change this behavior.

Reply:
I don't believe there was ever an explicit extension filters. I didn't remove it. 

If you'd like one instead of or in addition to this I'm happy to change how this behaves. It could be regexes instead for example.